test/langtools/tools/javac/diags/examples/NotApplicableTypes.java

```diff
@@ -21,8 +21,8 @@
  * questions.
```

Comment:
The key does not exist any, per my understanding. I would suggest to simply delete the file.